css/mobile-first.css

```diff
@@ -88,21 +96,25 @@ header {
   padding: 8px;
   border-radius: 4px;
   display: inline-block;
```

Comment:
alternatively, you can `display: block` if you don't have anything coming after the btn that needs to be inline with it.